utils/aplay/aplay.c

```diff
@@ -1073,6 +1076,28 @@ int main(int argc, char *argv[]) {
 			syslog = true;
 			break;
 
+		case 9 /* --loglevel */ : {
```

Comment:
How about moving this to the second `getopt_long()`? In case of invalid value there is an `error()` call which will happen before the log is initialized. It's not a big issues, but maybe it will be better to call `error()` after `log_open()`.